README.md

```diff
@@ -109,13 +109,27 @@ You probably need to set some environment variables for the client to be useful.
 
 ## Troubleshooting
 
+### I'm getting compilation errors
+
+Make sure you are using NodeJS 12. This may involve installing an LTS version from a package manager rather than the latest stable version.
+
+### The server can't create an HID device
+
+Run the server as root.
```

Comment:
As you noted, there actually is a better way:

https://github.com/node-hid/node-hid#udev-device-permissions